style.css

```diff
@@ -904,11 +988,51 @@
     background-color: #ffff;
   }
 
+  #pop-up-window {
+    display: none;
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    z-index: 9999;
+    width: 100%;
+    height: 100%;
+    backdrop-filter: blur(10px);
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .pop-up-container {
+    align-self: center;
+    height: 100%;
+    width: 100%;
+    background-color: white;
+    padding: 3%;
+  }
+
+  .grid-container-popup {
+    display: flex;
+    flex-direction: column;
+    max-width: 85%;
+    gap: 24px;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    width: 100%;
+    height: 100%;
+    background-color: #ffff;
+  }
```

Comment:
- [ ] Great job implementing the popup for the desktop version :+1: However you need to make a little change to make it look like the Figma design. Kindly follow these steps to fix that :
  - Change the height to `height: 80%;` to reduce the height of the popup
  - Change the gap to `gap: 10px:` to reduce the space between the elements
  - Add `overflow-y: scroll;` to this class to make it scroll when the content is overflow

### Your Design :point_down: 
![Screenshot from 2023-04-13 17-34-47](https://user-images.githubusercontent.com/91142432/231839523-73e0c11d-2a49-4b75-a81a-dee394ad70bf.png)

### What it supposed to look like :point_down: 
![](https://user-images.githubusercontent.com/104968236/231757248-b644c67b-8b0b-4859-aad7-050f8a5d5b5d.png)

Reply:
I'm concerned because that was my last code review

Reply:
making the last styling changes

![image](https://user-images.githubusercontent.com/111823606/231855758-65a0a350-d1ef-4af9-b4b0-81784c8e01c2.png)